applications/FluidDynamicsApplication/custom_processes/mass_conservation_check_process.cpp

```diff
@@ -0,0 +1,265 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    under supervision of Ruben Zorrilla
+//
+//
+
+// System includes
+#include <iostream>
+#include <fstream>
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
```

Comment:
You are including what you have already included in the header! The majority of these includes can be removed by directly including the header of this class.

Reply:
Done. Removed.